packages/fx-core/src/plugins/resource/frontend/ops/deploy.ts

```diff
@@ -64,8 +65,9 @@ export class FrontendDeployment {
   }
 
   public static async skipBuild(): Promise<void> {
-    const progressHandler = ProgressHelper.deployProgress;
     Logger.info(Messages.SkipBuild());
+
+    const progressHandler = ProgressHelper.deployProgress;
```

Comment:
I like this kind of adjustment, it makes the code more compact even it has no function impact and no line reduce.